src/tracing/index.js

```diff
@@ -49,7 +49,7 @@ function shouldEnableTracing() {
 
 
 function shouldEnableAutomaticTracing() {
-  if (config.tracing && config.tracing.enabled === false) {
+  if (config.tracing && config.tracing.enabled === false && exports.supportedVersion(process.versions.node)) {
```

Comment:
In this case we should log a different message.

Reply:
We can log a message but it would be better if we warned on install via `engine`: https://docs.npmjs.com/files/package.json#engines

Reply:
Nope, we shouldn't warn on install. The sensor even works on Node.js 0.12. IMHO we should consider automatic tracing an optional feature. This is how we treated it before. Also, correctly interpreting the engine field is harder than interpreting a good log message.

Reply:
Ok I'll update the log message.